### DISCUSSION TITLE
This Deployment has been disabled.

### DISCUSSION THREAD

**Discussion by Rajesh-Royal** [vercel/community#5911] [Help] [answered]
### Summary

Hi team, My site which is deployed on vercel is disabled, probably vercel system/algo thinks it is a spam site. But It is just a UI clone of Netflix, nothing else. The login signup forms of the UI Clone do not work they are just to sit there and show the user the UI.

I use this UI Clone to showcase my frontend skills to recruiteers. Please enable it.
```
This Deployment has been disabled.

Your connection is working correctly.

Vercel is working correctly.

451: UNAVAILABLE_FOR_LEGAL_REASONS
Code: DEPLOYMENT_DISABLED
ID: bom1::6qz2t-1708751109275-635e81f06bc9

If you are a visitor, contact the website owner or try again later.
If you are the owner, please [contact our support team](https://vercel.com/help).

```
![image](https://github.com/vercel/community/assets/24524924/bf525ee0-f1df-41d4-8850-8c1eee45bd9b)

Below is the details of deployment - 

451: UNAVAILABLE_FOR_LEGAL_REASONS
Code: DEPLOYMENT_DISABLED
ID: bom1::6qz2t-1708751109275-635e81f06bc9
Url: https://netflix-frontend-clone-theta.vercel.app/

### Example

https://netflix-frontend-clone-theta.vercel.app/

### Steps to Reproduce

Visit the production URL which is given above:

**Comment by swarnava** [comment, marked as answer]
Hi,

Please email us at abuse@vercel.com

**Reply by Rajesh-Royal** [reply]
Sent an Email. Thanks @swarnava
